### ISSUE TITLE
Accessing nova backend throws exception

### ISSUE THREAD

**Issue by lukas-cierra** [laravel/nova-issues#2933]
- Laravel Version: 8.8.0
- Nova Version: 3.11.1
- PHP Version: 7.3.20

### Description:
When accessing the nova backend on `/admin` i get an exception:

`Method Illuminate\View\View::__toString() must not throw an exception, caught ErrorException: htmlspecialchars() expects parameter 1 to be string, array given (View: /Users/lukas/codebase/test-project/vendor/laravel/nova/resources/views/resources/navigation.blade.php) {"userId":11,"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Method Illuminate\\View\\View::__toString() must not throw an exception, caught ErrorException: htmlspecialchars() expects parameter 1 to be string, array given (View: /Users/lukas/codebase/test-project/vendor/laravel/nova/resources/views/resources/navigation.blade.php) at /Users/lukas/codebase/test-project/storage/framework/views/7c34173907a3ddd752cc4988b38696bc43498ab3.php:0)`

I found out that the problem is a Nova resource called in this example `Order`. If I delete the resource, the backend works. If i rename the resource, it also works.
I also deleted the resource and created a new one, but the problem still exists.

**Comment by crynobone** [comment]
Can you share the affecting `Order` file here.

**Comment by lukas-cierra** [comment]
Here is the `Order` file. It is very basic right now:

```
<?php declare(strict_types=1);

namespace App\Nova;

use Illuminate\Http\Request;
use Laravel\Nova\Fields\BelongsTo;
use Laravel\Nova\Fields\ID;

class Order extends Resource
{
    /**
     * The model the resource corresponds to.
     *
     * @var string
     */
    public static $model = \App\Models\Order::class;

    /**
     * The single value that should be used to represent the resource when being displayed.
     *
     * @var string
     */
    public static $title = 'id';

    /**
     * The columns that should be searched.
     *
     * @var array
     */
    public static $search = [
        'id',
    ];

    /**
     * Get the fields displayed by the resource.
     */
    public function fields(Request $request): array
    {
        return [
            ID::make(__('ID'), 'id')->sortable(),

            BelongsTo::make(__('User')),

            BelongsTo::make(__('Company')),

            BelongsTo::make(__('Module')),
        ];
    }

    /**
     * Get the cards available for the request.
     */
    public function cards(Request $request): array
    {
        return [];
    }

    /**
     * Get the filters available for the resource.
     */
    public function filters(Request $request): array
    {
        return [];
    }

    /**
     * Get the lenses available for the resource.
     */
    public function lenses(Request $request): array
    {
        return [];
    }

    /**
     * Get the actions available for the resource.
     */
    public function actions(Request $request): array
    {
        return [];
    }
}

```

**Comment by crynobone** [comment]
I can't replicate the issue. If you check `resources/views/resources/navigation.blade.php` there's only 3 locations where the code that actually converted to `string` from Resource and those by right should always be `string` and not `array` (as reported here).

**Comment by crynobone** [comment]
Fixed in `v3.12.0` via laravel/nova#1003
